static/js/factories/learning_resources.js

```diff
@@ -133,3 +134,30 @@ export const makeLearningResource = (objectType: string) => {
     return R.merge({ object_type: LR_TYPE_USERLIST }, makeUserList())
   }
 }
+
+const formatFavorite = contentType => resource => ({
+  content_data: resource,
+  content_type: contentType
+})
+
+const makeFavorite = obj => ({ ...obj, is_favorite: true })
+
+export const makeFavoritesResponse = () => {
+  const courses = R.times(makeCourse, 3).map(makeFavorite)
+  const programs = R.times(makeProgram, 3).map(makeFavorite)
+  const bootcamps = R.times(makeBootcamp, 3).map(makeFavorite)
+
+  return [
+    ...courses.map(formatFavorite(LR_TYPE_COURSE)),
+    ...programs.map(formatFavorite(LR_TYPE_PROGRAM)),
+    ...bootcamps.map(formatFavorite(LR_TYPE_BOOTCAMP))
+  ]
+}
+
+export const courseListResponse = (list: Array<LearningResource>) => ({
```

Comment:
This only seems to be used for tests, maybe it should go in `test_utils.js` instead?

Reply:
sure